components/vf-link-list/vf-link-list--easy.njk

```diff
@@ -0,0 +1,25 @@
+<div class="vf-links {%- if component_modifier %} {{ component_modifier }}{% endif %}">
```

Comment:
Does this need to be a separate template? (Recalling that for wordpress one template file is better)

Reply:
Yeah. It does really - I'll amend, update, and push